bashtop

```diff
@@ -132,6 +137,13 @@ menu_quit_selected=(
 "║═╬╗║ ║ ║ ║  "
 "╚═╝╚╚═╝ ╩ ╩  ")
 
+menu_update_selected=(
```

Comment:
Should be double lined to follow the look of selected item

Reply:
ah yes ! I had done some research to find the same type of fonts I came across the 
**Calvin S** type fonts but I couldn't find the double line ... If you know the type of font, send me the link  i 'll fix that

Reply:
It's Calvin S but uppercase :)
http://patorjk.com/software/taag/#p=display&f=Calvin%20S&t=UPDATE

Reply:
thanks  :100: